atomic_reactor/plugins/post_store_metadata_in_osv3.py

```diff
@@ -55,13 +58,23 @@ def run(self):
             return
         self.log.info("build id = %s", build_id)
 
-        api_url = urljoin(self.url, "/osapi/v1beta1/")
-        oauth_url = urljoin(self.url, "/oauth/authorize")  # MUST NOT END WITH SLASH
-
         # initial setup will use host based auth: apache will be set to accept everything
         # from specific IP and will set specific X-Remote-User for such requests
-        # FIXME: use OSBS here
-        o = Openshift(api_url, oauth_url, None, use_auth=self.use_auth, verify_ssl=self.verify_ssl)
+        with tempfile.NamedTemporaryFile(mode='w+') as f:
```

Comment:
I think we don't need to store anything on disk to make this work, something like this should be possible:
```
Configuration(conf_file=None, conf_section=None, cli_args=None, openshift_uri="...", use_auth=..., ver...)
```
If this won't work, I'm in favor of patching osbs.